README.md

```diff
@@ -1,5 +1,28 @@
 # TARL Bulk Tools
 
+This guide will demonstrate two methods of running this tool locally - Docker and XAMPP.
+
+## Running with Docker
+
+If you already have [Docker](https://docs.docker.com/get-docker/) installed/running on your machine then run the following command from the root directory of this project to start this tool:
+
+```
+docker run -d -p 80:80 -v "$PWD":/var/www/html php:7.2-apache
```

Comment:
Might be worth an explanation that this mounts your local folder as a volume in the docker guest machine and so any changes locally will be reflected in the docker machine?

Also the user will need to dip back into the xampp instructions at a later step to complete the user config setup?

Reply:
Good points.  I don't want to overcomplicate with too detailed an explanation of Docker, but I will add something to cover it...